src/language-server/project/client.ts

```diff
@@ -201,20 +201,7 @@ export class GraphQLClientProject extends GraphQLInternalProject {
     super({ config, configFolderURI, loadingHandler, clientIdentity });
     this.serviceID = config.graph;
 
-    /**
-     * This function is used in the Array.filter function below it to remove any .env files and config files.
-     * If there are 0 files remaining after removing those files, we should warn the user that their config
-     * may be wrong. We shouldn't throw an error here, since they could just be initially setting up a project
-     * and there's no way to know for sure that there _should_ be files.
-     */
-    const filterConfigAndEnvFiles = (path: string) =>
-      !(
-        path.includes("apollo.config") ||
-        path.includes(".env") ||
-        (config.configURI && path === config.configURI.fsPath)
-      );
-
-    if (this.allIncludedFiles().filter(filterConfigAndEnvFiles).length === 0) {
```

Comment:
Since `this.fileSet` doesn't contain config files anymore, this is not necessary anymore.